src/components/main/featured/Featured.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import './featured-styles.scss'
+
+const Featured = () => {
+  return (
+    <div className="grid-wrapper">
+      <div className="subheader-container">
+        <h2 style={{fontSize: "2.375rem"}}>
+          Featured Items
+        </h2>
+      </div>
+      <div className="featured-content-container">
+        <div className="col">
+          <a href="#" style={{textDecoration: 'none'}}>
+            <img src={"https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=80"} />
+            <div className="featured-description">
+              <div className="featured-name">
+                <b>Product Name</b>
+              </div>
+              <div className="featured-rating">
+                {/* Still need consider the rating system */}
+                <p>&starf;&starf;&starf;&starf;&starf;</p>
```

Comment:
These star code thingies weren't rendering for me on my end. Im not sure why, but let me know if it was supposed to work cus maybe there was something funky going on when I pulled the code! Or maybe there's a browser compatibility thing we might need to consider.